tests/sentry/workflow_engine/migration_helpers/test_migrate_alert_rule.py

```diff
@@ -219,6 +217,8 @@ def assert_alert_rule_trigger_action_migrated(alert_rule_trigger_action, action_
             assert action.data == {
                 "priority": alert_rule_trigger_action.sentry_app_config["priority"],
             }
+    else:
+        assert action.data == {}
```

Comment:
nit: it might be better for us to break down this assertion into smaller bits per test rather than having a larger assertion test. The reason for that, is it can be a little difficult to debug test failures from larger methods like this. also, if the `DataConditionGroupAction` assertion fails, we won't know if the action.data is setup correctly because the test would fail before trying this assertion.